code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -580,39 +580,49 @@
 		if(tox_loss)
 			to_heal = min(10, tox_loss)
 			adjustToxLoss(0 - to_heal)
-			blood_used += round(to_heal * 1.2)
+			blood_used += round(to_heal * 0.25)
 		if(oxy_loss)
 			to_heal = min(10, oxy_loss)
 			adjustOxyLoss(0 - to_heal)
-			blood_used += round(to_heal * 1.2)
+			blood_used += round(to_heal * 0.25)
```

Comment:
These buffs look undocumented?

Reply:
oh, yeah. these had to be changed because it would drain all of your usable blood in a couple like two seconds, since i fixed the blood drain thing. since this was never player-facing (unless you had extremely low blood to hit the limit), does it need to go in the changelog?

Reply:
Yes, please.